core/sqf/src/tm/tmddlrequests.cpp

```diff
@@ -39,13 +40,24 @@ JNIEXPORT void JNICALL Java_org_apache_hadoop_hbase_client_transactional_RMInter
    char la_tbldesc[TM_MAX_DDLREQUEST_STRING];
    char la_tblname[TM_MAX_DDLREQUEST_STRING];
    char* str_key;
-   str_key = new char[TM_MAX_DDLREQUEST_STRING];
+   if(pv_keyLength <= 0) 
+   {
+     cout << "createTableReq bad input pv_keyLength, abort" << endl;
+     abort();
+   }
+   str_key = new char[pv_keyLength];
```

Comment:
@prashanth-vasudev , I realize I have to wait for TRAFODION-2148 check in first to do length checking. Currently, even the length is too long, there is no way to return error and abort the changes done before.
So I will wait for 2148 to be merged first.